pandemic/model.py

```diff
@@ -393,430 +400,234 @@ def pandemic_multiple_time_steps(
-  out_csv.to_csv(outpath + f'pandemic_output_aggregated.csv', 
-                float_format='%.2f', 
-                na_rep="NAN!")
-  presence_cols_monthly =  [c for c in presence_cols 
-                            if len(c.split(' ')[-1]) > 5]
-  presence_cols_annual = [c for c in presence_cols 
-                          if c not in presence_cols_monthly]
-  agg_prob_cols_annual = [c for c in formatted_geojson.columns 
-                          if c.startswith('Agg')]
-  
-  presence_d = create_feature_dict(geojson_obj = formatted_geojson,
-                                   column_list = presence_cols_annual,
-                                   chars_to_strip = 'Presence ')
-  agg_prob_d = create_feature_dict(geojson_obj = formatted_geojson,
-                                   column_list = agg_prob_cols_annual,
-                                   chars_to_strip = 'Agg Prob Intro ')
-  new_gdf = add_dict_to_geojson(geojson_obj = formatted_geojson,
-                              new_col_name = 'Presence',
-                              dictionary_obj = presence_d)
-  new_gdf = add_dict_to_geojson(geojson_obj = new_gdf,
-                              new_col_name = 'Agg Prob Intro',
-                              dictionary_obj = agg_prob_d)
-  cols_to_drop = [c for c in new_gdf.columns if
-                  c in presence_cols_monthly or
-                  c.startswith('Probability')]
-
-  sm_gdf = new_gdf.drop(cols_to_drop, axis=1)
-  sm_gdf.to_file(outpath + f'pandemic_output_aggregated_select.geojson', 
-                  driver='GeoJSON')
-  sm_csv = pd.DataFrame(sm_gdf)
-  sm_csv.drop(['geometry'], axis=1, inplace=True)
-  sm_csv.to_csv(outpath + f'pandemic_output_aggregated_select.csv', 
-                float_format='%.2f', 
-                na_rep="NAN!")
-
-# trade = np.array(
-#     [
-#         [
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#         ],
-#         [
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#             [[0, 500, 15], [50, 0, 10], [20, 30, 0]],
-#         ],
-#     ]
-# )
-# trade[1, :, :, :] = trade[1, :, :, :] * 1.1
-
-# trade = np.array([[0, 500, 15], [50, 0, 10], [20, 30, 0]])
-# distances = np.array([[1, 5000, 105000], [5000, 1, 7500], [10500, 7500, 1]])
-# locations = pd.DataFrame(
-#     {
-#         "name": ["United States", "China", "Brazil"],
-#         "Phytosanitary Capacity": [0.00, 0.00, 0.00],
-#         "Presence": [True, False, True],
-#         "Host Percent Area": [0.25, 0.50, 0.35],
-#     }
-# )
-#%%#
-data_dir = data_dir
-countries = geopandas.read_file(
-    countries_path,
-    driver='GPKG')
+with open(path_to_config_json) as json_file:
+    data = json.load(json_file)
```

Comment:
Would it be more clear to call this config instead of data? Might help distinguish between data inputs and configuration parameter inputs. Not a big deal though.